libexec/pyenv

```diff
@@ -40,7 +40,7 @@ else
 
     # Use a subshell to avoid changing the current path
     (
-    while [ -n "$path" ]; do
+    while [[ -n "$path" && $(echo "$path" | grep /) ]]; do
```

Comment:
This will return "pwd" if "$path" is /. At least it must not be the correct return value of the function.

Reply:
I think some fix like following will work sufficiently. Please consider adding tests for it.

'''
while [[ "$path" == *"/"* ]]; do
  cd "${path%/*}"
  local name="${path##*/}"
  [ -n "$name" ] || break
  path="$(resolve_link "$name" || true)"
done
'''

Reply:
Thanks! I'll check this

Reply:
Just checked it: it doesn't work your way at all. Will need to investigate some better way.